embedding/cpp/include/flutter_app.h

```diff
@@ -81,6 +81,16 @@ class FlutterApp : public flutter::PluginRegistry {
   // must be added to tizen-manifest.xml file.
   bool is_top_level_ = false;
 
+  // The renderer type of the engine.
+  //
+  // Defaults to kEGL. If the profile is wearable, defaults to kEvasGL.
+#ifdef WEARABLE_PROFILE
+  FlutterDesktopRendererType renderer_type_ =
```

Comment:
The type `FlutterDesktopRendererType` is not expected to be exposed through the public interface of the embedding. I'll talk to you offline in detail.

Reply:
> Should these be fixed later as well?

Well, I think the cases you mentioned are implemented in that way on purpose (to directly access underlying native pointers) and thus won't need a fix. However, let me take a look again.

Reply:
`IPluginRegistry` is a C# version of C++'s [`PluginRegistry`](https://github.com/flutter-tizen/engine/blob/flutter-3.0.0-tizen/shell/platform/common/client_wrapper/include/flutter/plugin_registry.h), and its `GetRegistrarForPlugin` method is expected to return a raw `FlutterDesktopPluginRegistrar` handle.

That said, we can still provide a higher level API of `FlutterDesktopPluginRegistrar` in the C# side which provides the same functionality as C++'s [`PluginRegistrar`](https://github.com/flutter-tizen/engine/blob/flutter-3.0.0-tizen/shell/platform/common/client_wrapper/include/flutter/plugin_registrar.h) class. However, even if it's implemented, there will be no use for it.

Reply:
Also I was thinking of introducing a new public interface for `FlutterApp` that can be used to configure all (currently experimental) window-related properties (such as `window_offset_x_` and `is_window_transparent_`) and also the renderer type in this PR at once using a single struct. I don't have a concrete plan yet. You may think about it if interested (maybe later).